pkg/tiller/kind_sorter.go

```diff
@@ -123,5 +123,9 @@ func (k *kindSorter) Less(i, j int) bool {
 	if !ok {
 		return true
 	}
+	if first == second {
+		// same kind so sub sort alphanumeric
+		return a.name < b.name
+	}
```

Comment:
Note that if either kind is unknown, the ordering is still contingent on the input order. Is that acceptable? Perhaps we should sort by name if both types are unknown.